exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -184,6 +189,7 @@ func (prwe *prwExporter) handleExport(ctx context.Context, tsMap map[string]*pro
 
 // export sends a Snappy-compressed WriteRequest containing TimeSeries to a remote write endpoint in order
 func (prwe *prwExporter) export(ctx context.Context, requests []*prompb.WriteRequest) error {
+	prwe.settings.Logger.Debug(fmt.Sprintf("Preparing to make HTTP request to: %s", prwe.clientSettings.Endpoint))
```

Comment:
can we use a zap.Field here instead of using `fmt.Sprintf` which will run regardless if debug is enabled or not?